README.md

```diff
@@ -38,6 +38,7 @@ Other Style Guides
   1. [Accessors](#accessors)
   1. [Events](#events)
   1. [jQuery](#jquery)
+  1. [Line length](#line-length)
```

Comment:
Instead of adding new section, perhaps you can add your new rule to the `Whitespace` section